virt-v2v/pkg/customize/rhel.go

```diff
@@ -134,3 +119,19 @@ func addLuksKeysToCustomize(extraArgs *[]string) error {
 
 	return nil
 }
+
+func addRhelDynamicScripts(extraArgs *[]string, dir string) error {
+	dynamicScripts, err := getScriptsWithRegex(dir, global.LINUX_DYNAMIC_REGEX)
```

Comment:
each time we add scripts we need to sort them, so they will run in order, `10-dothing.sh `before `20-something.sh` i am missing where we do it

Reply:
It is already sorted in the filesystem so when we do the `os.ReadDir` we get it in the order